packages/discovery-provider/src/queries/search_es.py

```diff
@@ -539,7 +540,7 @@ def track_dsl(
             {
                 "bool": {
                     "should": [
-                        {"term": {"downloadable": {"value": False}}},
+                        {"term": {"downloadable": {"value": True}}},
                         {"term": {"has_stems": {"value": True}}},
                     ]
                 }
```

Comment:
is there a reason that we are keeping both the `only_downloadable` and `only_with_downloads`?
looking at the `only_purchaseable` one below is seems like `only_with_downloads` would cover both cases by itself?

Reply:
yeah great question, lets sync quickly cause that also confused me

Reply:
@sliptype when you're back lets discuss.